pkg/pidfile/pidfile_test.go

```diff
@@ -0,0 +1,44 @@
+package pidfile
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestCreateFile(t *testing.T) {
```

Comment:
this entire test case is covered in the TestRemove test case below. I believe in such cases we omit the extra method. https://github.com/docker/docker/blob/082b7726e22c4e7e161cb715158675ed24fc3e4c/integration-cli/docker_cli_build_test.go#L1816-1818

Or you may wanna create a helper method from the common code in which you pass `t` that does the common job and this case + TestRemoveCreatedFile can use that.